polyfill/test/zoneddatetime.mjs

```diff
@@ -1291,6 +1423,71 @@ describe('ZonedDateTime', () => {
     });
   });
 
+  describe('math order of operations and options', () => {
+    const breakoutUnits = (op, zdt, d, options) =>
+      zdt[op]({ years: d.years }, options)
+        [op]({ months: d.months }, options)
+        [op]({ weeks: d.weeks }, options)
+        [op]({ days: d.days }, options)
+        [op](
+          {
+            hours: d.hours,
+            minutes: d.minutes,
+            seconds: d.seconds,
+            milliseconds: d.milliseconds,
+            microseconds: d.microseconds,
+            nanoseconds: d.nanoseconds
+          },
+
+          options
+        );
+
+    it('order of operations: add / none', () => {
+      const zdt = ZonedDateTime.from('2020-01-31T00:00-08:00[America/Los_Angeles]');
+      const d = Temporal.Duration.from({ months: 1, days: 1 });
+      const options = undefined;
+      const result = zdt.add(d, options);
+      equal(result.toString(), '2020-03-01T00:00:00-08:00[America/Los_Angeles]');
+      equal(breakoutUnits('add', zdt, d, options).toString(), result.toString());
+    });
+    it('order of operations: add / constrain', () => {
+      const zdt = ZonedDateTime.from('2020-01-31T00:00-08:00[America/Los_Angeles]');
+      const d = Temporal.Duration.from({ months: 1, days: 1 });
+      const options = { overflow: 'constrain' };
+      const result = zdt.add(d, options);
+      equal(result.toString(), '2020-03-01T00:00:00-08:00[America/Los_Angeles]');
+      equal(breakoutUnits('add', zdt, d, options).toString(), result.toString());
+    });
+    it('order of operations: add / reject', () => {
+      const zdt = ZonedDateTime.from('2020-01-31T00:00-08:00[America/Los_Angeles]');
+      const d = Temporal.Duration.from({ months: 1, days: 1 });
+      const options = { overflow: 'reject' };
+      throws(() => zdt.add(d, options), RangeError);
+    });
+    it.skip('order of operations: subtract / none', () => {
```

Comment:
Thanks for these tests. Will be helpful in validating #993 fix.

Reply:
I don't know if you remember, but you wrote them... :smile:

Reply:
Haha, I totally forgot. You should have taken the credit. I would have never remembered! ;-)